src/services/services.ts

```diff
@@ -3903,7 +3903,8 @@ module ts {
                 }
 
                 // if this symbol is visible from its parent container, e.g. exported, then bail out
-                if (symbol.parent) {
+                // if symbol correspond to the union property - bail out
+                if (symbol.parent || (symbol.getFlags() & SymbolFlags.UnionProperty)) {

```

Comment:
Why is symbol.parent not defined here? Doesn't the checker give union properties a parent?